src/main/resources/blockly/js/mcreator_blocks.js

```diff
@@ -280,6 +280,16 @@ Blockly.Blocks['java_code'] = {
     }
 };
 
+Blockly.Blocks['java_code_get'] = {
+    init: function () {
+        this.appendDummyInput()
+            .appendField(javabridge.t("blockly.block.java_code"))
+            .appendField(new Blockly.FieldTextInput("/*code*/"), 'CODE');

```

Comment:
The default value should not be a comment but say `null`, as the output block requires a value, not a statement.

Reply:
:heavy_check_mark: Done
But I think <code>null</code> is not fine there, that's why output code snippet returns an error message instead of appending <code>null</code>.

Reply:
I meant null not as string, but null value. null is the only universtal return type. Output block must return some kind of value and by "returning" null, you indicate this.